deta/__init__.py

```diff
@@ -16,11 +16,10 @@
 except Exception:
     pass
 
+__version__ = 0.4
```

Comment:
the current version is `0.7`. also change it in setup.py, for some reason we didn't commit it there too

Reply:
Will do!